sdk/web-pubsub/web-pubsub/test/hubs.spec.ts

```diff
@@ -6,9 +6,11 @@ import { WebPubSubServiceClient, AzureKeyCredential } from "../src";
 import { assert } from "chai";
 import environmentSetup from "./testEnv";
 import { FullOperationResponse } from "@azure/core-client";
+import { DefaultAzureCredential } from "@azure/identity";
 
 describe("HubClient", function() {
   let recorder: Recorder;
+  this.timeout(30000);
```

Comment:
😢 Can we at least only do this when we're running in live or record mode? Slow playback tests taking forever to fail is sad